nss-evening-cohort-24/pos-system-the-ronis
# Details Button (BASIC)

## Description
- Created API call for getItems and made a seperate files. Gets all the items. 

## Related Issue
#19 , #22 

## Motivation and Context
This is a necessary building block to creating the core functionality of the POS. 

## How Can This Be Tested?
- Pull PR branch to local machine.
- Stand web-server. 
- Make sure that when the 'Details' button on the 'Order' card is selected, the DOM is cleared and all items (FOR NOW), are rendered.

## Screenshots (if appropriate):

## Types of changes
